creusot-contracts/src/std/slice.rs

```diff
@@ -400,7 +399,7 @@ impl<'a, T> ShallowModel for IterMut<'a, T> {
     #[open(self)]
     #[trusted]
     #[ensures((^result)@.len() == (*result)@.len())]
-    fn shallow_model(self) -> Self::ShallowModelTy {
+    fn shallow_model(#[creusot::open_inv] self) -> Self::ShallowModelTy {
```

Comment:
This attribute does not have any effect during the translation of `03_std_iterators`, causing a cycle.